### PR TITLE
Port the renderPass resolve tests from GLSL to WGSL

### PR DESCRIPTION
**Pull request by Kangz**
PTAL! They fail on my machine but the GLSL tests fail in the same way.

### CODE REVIEW

File: src/webgpu/api/operation/render_pass/resolve.spec.ts [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+      }
+    }
+
+    const encoder = t.device.createCommandEncoder();
+
+    const pass = encoder.beginRenderPass({
+      colorAttachments: renderPassColorAttachmentDescriptors,
+    });
+    pass.setPipeline(pipeline);
+    pass.draw(3, 1, 0, 0);
```
Comment (austinEng) [inline src/webgpu/api/operation/render_pass/resolve.spec.ts:154]: .draw(3)

**Review by austinEng — APPROVED** [review]
LGTM

**Review by kainino0x — APPROVED** [review]
For resolve.spec.ts I just looked at https://github.com/gpuweb/cts/compare/glsl-dependent..38901c374c07aeb2ec6d3085e2b7cf51e1596d0b#diff-096755e42b939395543fb17862b1704323eada522fc6c3874bf59e5cd17f4fa4 instead of the whole file again. LGTM

File: src/webgpu/api/operation/render_pass/resolve.spec.ts (hunk 1 above)
Comment (kainino0x) [reply src/webgpu/api/operation/render_pass/resolve.spec.ts:154]:
```suggestion
    pass.draw(3);
```

**Comment by Kangz** [comment]
Thanks for the link @kainino0x, I'll use post a similar link next times!

**Comment by Kangz** [comment]
Just to make sure, is this ready to be merged?

**Comment by kainino0x** [comment]
yes, had just left it open in case we decided to go the other route merging changes into the branch first.

**Pull request merged**